src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -29,6 +31,24 @@ public static void WriteNumber(bool indented)
             WriteSimpleValue(indented, "42");
         }
 
+        [Theory]
+        [InlineData("12E-3")]
+        [InlineData("12e+3")]
+        //        [InlineData("1.2e+13")] Enable after https://github.com/dotnet/corefx/issues/39139 resolved
+        //        [InlineData("1.2E-3")]
+        public static void WriteScientificNumber(string value)
+        {
+            var buffer = new ArrayBufferWriter<byte>(1024);
+            using (JsonDocument doc = JsonDocument.Parse($" [ {value} ]", s_options))
+            {
+                JsonElement target = doc.RootElement[0];
+
+                var writer = new Utf8JsonWriter(buffer);
```

Comment:
Utf8JsonWriter is IDisposable, so all of these writers should be in `using` blocks